About & Profile Pages

cahdeemer opened the pull request.

cahdeemer committed:
About & Profile Pages

mtedeschi commented on about.html:
```diff
@@ -0,0 +1,147 @@
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Mayor's Fund for Philadelphia</title>
```
Just a future note to have SEO friendly titles, like "About the Fund - The Mayor's Fund for Philadelphia | A nonprofit that seeks to improve the quality of lives for all Philadelphians"

cahdeemer replied:
Got it. Done.

mtedeschi commented on about.html:
```diff
@@ -0,0 +1,147 @@
+  	</header>
+    
+    <main>
+    	<section class="firsthing">
```
Small detail: but as a best practice it's a good thing to get in the habit of naming classes or IDs something everyone can quickly recognize. This could be "jumbotron" or "hero".

cahdeemer replied:
Got it.  I was worried about interfering with existing Bootstrap classes.  I think there is a Bootstrap class called "Jumbotron."  In any case, I changed "firstthing" to "hero."

mtedeschi commented on about.html:
```diff
@@ -0,0 +1,147 @@
+    					<h4>Excellence</h4>
+    					<p>The Fund is committed to excellence in all that it does, and to always go the extra step for the residents of Philadelphia.</p>
+    				</div>
+    				<aside class="col-sm-4 history">
```
I would wrap the content inside `col-sm-4` with the `history` class, instead of putting it on the column (easier to style without affecting the column's styles).

cahdeemer replied:
Got it.  I created a separate div on which I put the history class.

mtedeschi commented on css/Sass/main.scss:
```diff
@@ -77,7 +84,10 @@ h1 {
 
-
+.history {
+	background-color: #e1e1e1;
```
We could move this out to a variable if we use it often.

mtedeschi commented on css/Sass/profile.scss:
```diff
@@ -0,0 +1,22 @@
+	h1 {
+		font-size: 48px;
+		position: initial;
+			small {
```
It looks like a lot of the same style is already defined in main.scss. Is this necessary?

cahdeemer replied:
Perhaps not.  The only thing that is different is the position (absolute in main; here it is initial).  There is probably a better way of handling both situations.

mtedeschi commented:
Overall, majority of this looks great. Just some small tweaks and things to think about.

cahdeemer committed:
Made Mike's edits

- added SEO text to page titles
- replaced “.firstthing” with “.hero”
- removed “.history” from columns and created <div class=“history”>
